docs/explanation/archive.rst

```diff
@@ -0,0 +1,348 @@
+for a given :ref:`ArchiveSeries` that are supported and maintained by :term:`Canonical`.
+
+:term:`Packages <Package>` in this *component* are mostly proprietary drivers for devices and similar.
+
+.. _ArchiveComponents_Universe:
+
+universe
+~~~~~~~~
+
+This *component* contains :term:`Open Source <Open Source Software>` :term:`Packages <Package>`
+for a given :ref:`ArchiveSeries` that are supported and maintained by the :term:`Ubuntu` community.
+
+.. _ArchiveComponents_Multiverse:
+
+multiverse
+~~~~~~~~~~
+
+This *component* contains :term:`Packages <Package>` (for a given :ref:`ArchiveSeries`) 
+of :term:`Closed Source Software` or :term:`Open Source Software` restricted by copyright
+or legal issues. These :term:`Packages <Package>` are maintained and supported by
+the :term:`Ubuntu` community, but because of the restrictions, patching bugs or updates 
+may not be possible.
+
+.. _ArchiveMirrors:
+
+Mirrors
+-------
+
+Every day, hundreds of thousands of people want to download & install :term:`Packages <Package>`
+from the *Ubuntu Package Archive*. To provide a decent :term:`User Experience`, the
+content of ``http://archive.ubuntu.com/ubuntu`` gets mirrored (replicated and kept
+in sync) by other servers to distribute network traffic, reduce latency, and provide
+redundancy, ensuring high availability and fault tolerance.
+
+You can find a complete list of officially recognized *Ubuntu Package Archive Mirrors*
+`here <https://launchpad.net/ubuntu/+archivemirrors>`_.
+
+.. note::
+
+    There also exist mirrors for the Ubuntu :term:`ISO` images (also called *"CD images"*,
+    because :term:`ISO` images which can be downloaded and burned to a CD to make
+    installation disks.)
+
+    You can find a complete list of officially recognized *Ubuntu CD Mirrors*
+    `here <https://launchpad.net/ubuntu/+cdmirrors>`_.
+    
+
+Country Mirrors
+~~~~~~~~~~~~~~~
+
+*Ubuntu Package Archive Mirrors* that provide a very reliable service in a country
+can request to be the official *Country Mirror* for that country. :term:`Ubuntu`
+installations are configured by default to use the *Country Mirror* for their selected country.
+
+*Country Mirrors* are accessible via the domain name format:
+  
+.. code:: text
+
+    <country-code>.archive.ubuntu.com
+
+You can see which mirror is the *Country Mirror* by doing a simple :term:`DNS` lookup. For example:
+
+.. tab-set::
+
+    .. tab-item:: Finland (FI)
+
+        .. code:: bash
+
+            dig fi.archive.ubuntu.com +noall +answer
+
+        .. code:: text
+
+            fi.archive.ubuntu.com.	332	IN	CNAME	mirrors.nic.funet.fi.
+            mirrors.nic.funet.fi.	332	IN	A	193.166.3.5
+
+        Therefore ``mirrors.nic.funet.fi`` is finlands *Country Mirror*.
+
+    .. tab-item:: Tunisia (TN)
+
+        Tunisia does not have any third-party mirrors in its country. Therefore the
+        Tunisia *Country Mirror* is just the main *Ubuntu Package Archive* server
```

Comment:
```suggestion
        Tunisia does not have any third-party mirrors in its country. Therefore the
        Tunisia country mirror is just the main Ubuntu package archive server
```

Reply:
I'd avoid use of the word "main" here to avoid new readers getting confused with the previous discussion of the *main* repository. Primary might work instead?